app/modules/rt-contact/class-rt-contact.php

```diff
@@ -206,6 +209,17 @@ function edit_view_filters($views){
 
 				$subsubsub[] = "<li><a href='edit.php?post_type=".rt_biz_get_contact_post_type().'&'.self::$user_category_taxonomy.'='.$term->slug."' class='".$current."'>".__( $term->name )."<span class='count'> (".count( $posts->posts ).')</span></a></li>';
 			}
+			$posts = new WP_Query( array(
+				'post_type' => $this->post_type,
+				'post_status' => 'trash',
+				'nopaging' => true,
+			) );
+			$current = '';
+			if ( $check_post_status && 'trash' == $_REQUEST['post_status'] ){
+				$current  = 'current';
+				$check_post_status = false;
+			}
+			$subsubsub[] = "<li><a href='edit.php?post_type=".rt_biz_get_contact_post_type()."&post_status=trash' class='".$current."'>".__( Trash )."<span class='count'> (". $posts->post_count .')</span></a></li>';
```

Comment:
Notice: Use of undefined constant Trash - assumed 'Trash' in /var/www/dummytest.com/htdocs/wp-content/plugins/rtbiz/app/modules/rt-contact/class-rt-contact.php on line 222